pkg/acsengine/defaults.go

```diff
@@ -130,8 +131,13 @@ var (
 		},
 		//KubernetesSpecConfig - Due to Chinese firewall issue, the default containers from google is blocked, use the Chinese local mirror instead
 		KubernetesSpecConfig: KubernetesSpecConfig{
-			KubernetesImageBase: "crproxy.trafficmanager.net:6000/google_containers/",
-			TillerImageBase:     "crproxy.trafficmanager.net:6000/kubernetes-helm/",
+			KubernetesImageBase:              "crproxy.trafficmanager.net:6000/google_containers/",
```

Comment:
Let's punt on modifying the China cloud spec, we usually defer to the China team to manage this. If you'd like, you could open up an issue to add ACI connector addon capability to China cloud and we can work with them on it.